--- src/text.c	2013-07-29 18:35:29.000000000 +0200
+++ src/text.c	2013-07-29 18:38:47.000000000 +0200
@@ -468,7 +468,8 @@
 
   /* check if previously loaded, otherwise load it now */
   if(!defaultfont[size]) {
-    snprintf(filename,128,"%s/G15/default-%.2i.fnt",G15FONT_DIR,size);
+    static const char *const WCFONTDIR = "data"; /* hardcoded font path for WC */
+    snprintf(filename,128,"%s/G15/default-%.2i.fnt",WCFONTDIR/*G15FONT_DIR*/,size);
     defaultfont[size] = g15r_loadG15Font(filename);
     if((defaultfont[size])==NULL) {
           fprintf(stderr,"libg15render: Unable to load font \"%s\"\n",filename);
